packages/react/src/components/DataTable/__tests__/TableToolbarSearch-test.js

```diff
@@ -1,75 +1,129 @@
 /**
- * Copyright IBM Corp. 2016, 2018
+ * Copyright IBM Corp. 2016, 2023
  *
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
 
 import React from 'react';
-import { mount } from 'enzyme';
-import { TableToolbarSearch } from '../';
-
-const prefix = 'cds';
-
-describe('DataTable.TableToolbarSearch', () => {
-  it('should render', () => {
-    const wrapper = mount(
-      <TableToolbarSearch
-        className="custom-class"
-        onChange={jest.fn()}
-        id="custom-id"
-      />
-    );
-    expect(wrapper).toMatchSnapshot();
-  });
-  it('should set an appropriate unique id', () => {
-    const wrapper = mount(
-      <TableToolbarSearch className="custom-class" onChange={jest.fn()} />
-    );
-    expect(typeof wrapper.find('Search').prop('id')).toBe('string');
-  });
-  it('should pass handleExpand as second parameter to onBlur/onFocus when provided', () => {
-    const onBlur = jest.fn().mockImplementation((event, handleExpand) => {
-      const { value } = event.target;
-      if (!value) {
-        handleExpand(event, false);
-      }
+import TableToolbarSearch from '../TableToolbarSearch';
+import userEvent from '@testing-library/user-event';
+import { render, screen } from '@testing-library/react';
+
+describe('TableToolbarSearch', () => {
+  describe('renders as expected - Component API', () => {
+    it('should render', () => {
+      const { container } = render(
+        <TableToolbarSearch
+          className="custom-class"
+          onChange={jest.fn()}
+          id="custom-id"
+        />
+      );
+
+      expect(container).toMatchSnapshot();
+    });
+
+    it('should support a custom `className` prop on the outermost element', () => {
+      const { container } = render(
+        <TableToolbarSearch className="custom-class" />
+      );
+
+      expect(container.firstChild).toHaveClass('custom-class');
+    });
+
+    it('should respect defaultExpanded prop', () => {
+      const { container } = render(<TableToolbarSearch />);
+
+      expect(container.firstChild).toHaveClass(
+        'cds--toolbar-search-container-expandable'
+      );
```

Comment:
should this test use the `defaultExpanded` prop?

Reply:
Good catch, updated and updated to the correct class so it actually tests something. 😅